brainscore_vision/benchmark_helpers/properties_common.py

```diff
@@ -65,8 +63,8 @@ def ceiling(self):
 
 
 @store(identifier_ignore=['model', 'in_rf'])
-def get_firing_rates(model_identifier, model, region, stimulus_identifier, number_of_trials, in_rf):
-    affine_transformation = firing_rates_affine(model_identifier=model_identifier, model=model, region=region)
+def get_firing_rates(model, region, stimulus_identifier, number_of_trials, in_rf):
+    affine_transformation = firing_rates_affine(model=model, region=region)
```

Comment:
this will run into issues with `store` where it will re-use a previous model's stored firing rates since there is no more `model_identifier` to separate them. When you revert this, can you add a comment to better explain?

Reply:
[a60bbc0](https://github.com/brain-score/brain-score/pull/415/commits/a60bbc002fa3c31a2f4c15781ddebf9034a210d3) reverts model_identifier removals, and [4715a93](https://github.com/brain-score/brain-score/pull/415/commits/4715a93ba16d15dee92f403da27e99379d332f14) adds a comment to explain.